### ISSUE TITLE
[Bug]: panic in `pkg/sql/compile` when run UT test

### ISSUE THREAD

**Issue by guguducken** [matrixorigin/matrixone#8232]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): 48539df
- Hardware parameters:
- OS type: Linux x64
- Others: 
```

### Actual Behavior

<img width="1217" alt="截屏2023-03-02 17 13 51" src="https://user-images.githubusercontent.com/22561920/222391984-cdf9dffd-db3d-4a42-827e-add6fb5f3658.png">

### Expected Behavior

no panic and FAIL

### Steps to Reproduce

```Markdown
make ut and see log
```

**Comment by guguducken** [comment]
> @guguducken pls add more info about
> 
> 1. OS type
> 2. commit id
> 3. failed case name

OS: Linux x64
commit id: 48539df
this panic occurs in the `/home/runner/work/matrixone/matrixone/matrixone/pkg/sql/compile/compile.init`

**Comment by daviszhen** [comment]
After discussion with @reusee , the compiler_context.go can be deleted。

**Comment by daviszhen** [comment]
But the compiler_context is referenced in some computation ut. It do not know how to fix it.

**Comment by nnsgmsone** [comment]
i will fix it

**Comment by nnsgmsone** [comment]
fixed by #8267
